torch/utils/tensorboard/writer.py

```diff
@@ -235,7 +252,7 @@ def _get_file_writer(self):
             else:
                 self.file_writer = FileWriter(logdir=self.log_dir, **self.kwargs)
             self.all_writers = {self.file_writer.get_logdir(): self.file_writer}
-        return self.file_writer
+        return self.file_writer 
```

Comment:
Nit - can you remove this whitespace?

Reply:
Interest, flake8 didn't complain about it.